specs/fjord/predeploys.md

```diff
@@ -16,4 +16,23 @@ Following the Fjord upgrade, three additional values used for L1 fee computation
 - costFastlzCoef
 - costTxSizeCoef
 
-These values are hard-coded constants in the `GasPriceOracle` contract.
+These values are hard-coded constants in the `GasPriceOracle` contract. The
+calculation follows the same formula outlined in the
+[Fjord L1-Cost fee changes (FastLZ estimator)](./exec-engine.md#fjord-l1-cost-fee-changes-fastlz-estimator)
+section.
+
+A new method is introduced: `getL1FeeUpperBound(uint256)`. This method returns an upper bound for the L1 fee
+for a given transaction size. It is provided for callers who wish to estimate L1 transaction costs in the
```

Comment:
I think you want to clarify this is for the *signed* transaction size?   Maybe also add that if it's unsigened, users should add 68 bytes to the length.

Reply:
thanks, fixed 👍 (made it unsigned)